lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -230,7 +231,7 @@ class ClassMetadataInfo implements ClassMetadata
      * hierarchy. If the entity is not part of a mapped inheritance hierarchy this is the same
      * as {@link $name}.
      *
-     * @var string
+     * @var class-string
```

Comment:
Here you write `@var class-string` but here https://github.com/doctrine/orm/pull/8409/commits/ef2260c211169286fff87949cf42adbe060be93e#diff-732e324167dd49e48b221477c3e0f6d7934f3eb5ec0970dbf1c06f6c7df15398R215-R216
you wrote used `@psalm-var`. Consistency ?

Reply:
done